027-FHIRPoweredHealthcare/readme.md

```diff
@@ -86,8 +77,7 @@ Your team's assistance is needed to implement this new event-driven FHIR ecosyst
   - Coach's guide to solutions for challenges, including tips/tricks.
 
 ## Contributors
+- Aruna Ranganathan (Microsoft)
```

Comment:
You can remove the (Microsoft) suffix since we don't have a customer in this list.